test/mocha/helper/login-helper.js

```diff
@@ -1,5 +1,8 @@
 'use strict';
 
+const student_name = process.env.STUDENT_NAME || 'schueler@schul-cloud.org';
+const password = process.env.PASSWORD || "Schulcloud1!";
```

Comment:
no passwords inside of code pls, fetch the value from a secrets file or similar instead...

Reply:
Seperate ticket pls, not relevant for redesign